### PR TITLE
updates to make dplus_us work

### PR DESCRIPTION
**Pull request by yeahme49**

### CODE REVIEW

File: resources/lib/dplay.py [hunk 1]
```diff
@@ -406,18 +407,18 @@ def get_current_episode_info(self, video_id):
...
+        data = json.loads(self.make_request(url, 'get', params=params, headers=self.site_headers))
         return data
 
     def get_next_episode_info(self, current_video_id):
         url = '{api_url}/content/videos/{video_id}/next'.format(api_url=self.api_url, video_id=current_video_id)
 
         params = {
             'algorithm': 'naturalOrder',
-            'include': 'genres,images,primaryChannel,show,show.images,contentPackages'
+            'include': 'genres,images,primaryChannel,show,show.images,contentPackages',
```
Comment (Dis90) [inline resources/lib/dplay.py:418]: Is this comma here by accident?

File: resources/lib/dplay.py [hunk 2]
```diff
@@ -540,21 +541,23 @@ def get_stream(self, video_id, video_type):
...
-            url = '{api_url}/playback/v3/videoPlaybackInfo/{video_id}'.format(api_url=self.api_url, video_id=video_id)
+            url = '{api_url}/playback/v3/videoPlaybackInfo'.format(api_url=self.api_url, video_id=video_id)
         else:
             if video_type == 'channel':
                 url = '{api_url}/playback/v2/channelPlaybackInfo/{video_id}'.format(api_url=self.api_url, video_id=video_id)
             else:
                 url = '{api_url}/playback/v2/videoPlaybackInfo/{video_id}'.format(api_url=self.api_url, video_id=video_id)
 
-        data_dict = json.loads(self.make_request(url, 'get', params=params, headers=self.site_headers))['data']
+        data_dict = json.loads(self.make_request(url, 'post', params=params, headers=self.site_headers, jsonPayload=jsonPayload))['data']
```
Comment (Dis90) [inline resources/lib/dplay.py:555]: Is parameter jsonPayload actually needed? Could this be payload=json.dumps(jsonPayload) ? Like here https://github.com/Dis90/plugin.video.discoveryplus/blob/ae08554d437f873419a3554ecfb7a02a6c990225/resources/lib/dplay.py#L268

Comment (yeahme49) [reply resources/lib/dplay.py]: changed it like your example and it works

File: resources/lib/dplay.py (hunk 1 above)
Comment (yeahme49) [reply resources/lib/dplay.py]: yes, removed

**Pull request merged**